src/analytics.ts

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+import { env, extensions, Extension, workspace } from "vscode";
+import * as https from "https";
+import * as querystring from "querystring";
+
+enum EventCategory {
+    Extension,
+	TODOs,
+}
+
+enum EventAction {
+    Activated,
+	Enabled,
+	Disabled
+}
+
+class Analytics {
+	private version: string;
+	private isDevelopment: boolean;
+	constructor() {
+		let packageJson = require('../../package.json');
+        this.version = packageJson.version;
+		this.isDevelopment = process.argv.indexOf("--type=extensionHost") >= 0;
+	}
+
+	logActivation() { this.log(EventCategory.Extension, EventAction.Activated); }
+	logShowTodosToggled(enabled: boolean) { this.log(EventCategory.TODOs, enabled ? EventAction.Enabled : EventAction.Disabled); }
+
+	private log(category: EventCategory, action: EventAction) {
+		let data = {
+			v: "1", // API Version.
+			tid: "UA-2201586-19",
+			cid: env.machineId,
```

Comment:
I actually say the same thing in the debugger and wondered if it was the dev host. I made a note to double-check what's really being sent from the release version but haven't had chance yet. I'll be sure to do it tonight!

If it doesn't work, I'll create a random ID and stash it in the extension context to use instead (and raise a bug with MS!)